pkg/doc/did/doc.go

```diff
@@ -360,12 +360,61 @@ func ParseDocument(data []byte) (*Doc, error) {
 	} else if raw == nil {
 		return nil, errors.New("document payload is not provided")
 	}
-	// validate did document
-	err = validate(data, raw.schemaLoader())
+
+	switch {
+	case strings.HasPrefix(raw.ID, "did:sov:"):
```

Comment:
This might not be 100% correct. This is more a case of AcaPy currently sending some document in DID exchange as `did:sov` ...
@fqutishat @Moopli

Reply:
yes

Reply:
@Moopli It might be safer to detect this case in DID Exchange code rather than here.